src/timer/timer_enable.cc

```diff
@@ -225,16 +225,20 @@ std::unique_ptr<TimerReport> TimerStatCollector::GetReport(TimerSection* section
     report->section_name_    = section->name_;
     report->timing_duration_ = last_merge_time_ - last_clear_time_;
     for (std::size_t i = 0; i < kTimerEntryBucketNum; ++i) {
+        const auto bucket_hits = entry.duration_buckets_[i].hits_;
+        report->has_data_ |= static_cast<bool>(bucket_hits);
```

Comment:
Can this be derived from bkt.size()?

Reply:
No, because the buckets are pre-allocated even if no hits.